### ISSUE TITLE
resample_hippdwi_to_template container

### ISSUE THREAD

**Issue by kaitj** [khanlab/hippunfold#215]
Was debugging with @Bradley-Karat and Mahdieh. Was encountering an error for missing `c3d` command. Noticed that the `resample_hippdwi_to_template` rule doesn't specify a container. Not sure if it needs to be specified?

**Comment by akhanf** [comment]
Yes that's a bug! Probably didn't notice it as those using it were running hippunfold from the single docker container.. Could use the autotop container here, if you're able to push a fix?
